### DISCUSSION TITLE
I want to trigger "angular-templatecache-webpack5-plugin" plugin in webpack after "copy-webpack-plugin" files are copied, how can I do this

### DISCUSSION THREAD

**Discussion by deepanshk** [webpack/webpack#18005] [General]
Here is the config how I want to trigger
 var templateCachePlugin = new AngularTemplateCacheWebpackPlugin({
            source: "build/dev/tmp/**/*.html",
            standalone: true,
            module: "templates-main",
            root: "../tmp/",
            outputFilename: "templates.js"
});

Objective here is once all files are copied, it will create a template.js file

**Comment by alexander-akait** [comment]
Please open an issue in `angular-templatecache-webpack5-plugin` repo and ask about set a stage/the hooks
